subsystem/src/main/java/org/wildfly/subsystem/resource/PathElementProvider.java

```diff
@@ -11,7 +11,9 @@
 /**
  * Provides a {@link PathElement}.
  * @author Paul Ferraro
+ * @deprecated To be removed without replacement
  */
+@Deprecated(forRemoval = true)
```

Comment:
@pferraro Could you add the since attribute for this deprecation?

@darranl added a relevant comment about its usage at https://github.com/wildfly/wildfly-core/pull/6084#discussion_r1696625630, I also think it is worth using it since it will provide information for future removal, although you can get it from the commit history the "since" makes more sense. Do you also agree? We can start getting it as a rule

Reply:
Done.

Reply:
@pferraro FYI, there are more than one @Deprecated annotation that needs to be adjusted with the "since", could you take another look?